lib/Wikia/src/Service/User/Permissions/PermissionsServiceImpl.php

```diff
@@ -88,6 +89,27 @@ public function getAutomaticGroups( \User $user, $recacheAutomaticGroups = false
 	}
 
 	/**
+	 * Returns list of access control groups that need to be added based on the explicit user groups
+	 *
+	 * @param \User $user
+	 * @return array
+	 */
+	private function getAutomaticAccessControlGroups( \User $user ) {
+		if ($user->getId()) {
+			$explicitGroups = $this->getExplicitGlobalGroups($user);
```

Comment:
You use different formatting i.e. ($user) instead of ( $user ).

Is this something that we care about?